src/gateway/kong-manager/consumer-groups.md

```diff
@@ -0,0 +1,96 @@
+
+## Set up a consumer group with consumers
+
+In this section, you will create the Bronze tier consumer group with a consumer assigned to the group.
+
+1. Open the **default** workspace.
+1. From the menu, open **Consumers**, then click the **Groups** tab.
+1. Click **New Consumer Groups**.
+1. Enter `Bronze` for the consumer group name and click **Create**.
+1. From the menu, click **Consumers**, then click **New Consumer**.
+1. Enter a **Username** and **Custom ID**. For this example, you can use `Amal` for each field.
+1. Click **Create**.
+1. From the menu, click the **Groups** tab.
+1. Click the `Bronze` consumer group you just created.
+1. Click **Consumers** and **Add consumers** to add the `Amal` consumer you created to the `Bronze` consumer group.
+
+## Configure the Rate Limiting Advanced plugin for all consumers
+
+In this section, you will configure the Rate Limiting Advanced plugin to set the rate limit to 5 requests every 30 seconds for all consumers.
+
+1. Open the **default** workspace.
+1. From the menu, open **Plugins**, then click **Install Plugin**.
+1. Find the **Rate Limiting** plugin, then click **Enable**.
+1. Apply the plugin as **Global**, which means the rate limiting applies to all requests, including every service and route in the workspace.
+
+    If you switched it to **Scoped**, the rate limiting would apply the plugin to only one service, route, or consumer.
+
+    By default, the plugin is automatically enabled when the form is submitted.
+    You can also toggle the **This plugin is Enabled** button to configure the plugin without enabling it.
+    For this example, keep the plugin enabled.
+1. Complete only the following fields with the following parameters.
+    1. data config.limit: `5`
+    1. data config.window_size: `30`
+    1. data config.window_type: `sliding`
+    1. data config.retry_after_jitter_max: `0`
+    1. data config.enforce_consumer_groups: `true` 
+   
+
+    Besides the above fields, there may be others populated with default values. For this example, leave the rest of the fields as they are.
+1. Click **Install**.
+
+## Configure rate limiting for consumer groups
+
+In this section, you will configure the Rate Limiting Advanced plugin to set the rate limit to 6 requests every 30 seconds for only consumers in the Bronze tier.
+
+1. Open the **default** workspace.
+1. From the menu, open **Consumers**, then click the **Groups** tab.
+1. Click the `Bronze` consumer group you just created.
+1. Click the **Policy** tab.
+1. Complete only the following fields with the following parameters.
+    1. data config.limit: `6`
+    1. data config.window_size: `30`
+    1. data config.window_type: `sliding`
+    1. data config.retry_after_jitter_max: `0`
+1. Click **Save**.
+
+## Validate consumer group rate limiting
+
+1. Enter `https://localhost:8000/mock` in your browser address bar, then refresh your browser six times.
+    After the 6th request, you’ll receive an error message.
```

Comment:
This won't do anything if you don't specify a consumer credential - the gateway has no idea if you're some specific consumer. You would need to do this first: https://docs.konghq.com/gateway/latest/get-started/key-authentication/

In the interest of time, I'd just take the validation section out for now - especially since I can't even get the feature to work at the moment.